src/main/java/ch/njol/skript/Metrics.java

```diff
@@ -28,1010 +29,1013 @@
 
 /**
  * bStats collects some data for plugin authors.
- *
+ * <p>

```

Comment:
Comes from Bstats, do not edit this file unless absolutely necessary.

Reply:
Sure, gonna rollback too